src/api.cc

```diff
@@ -346,19 +346,18 @@ static std::vector<Uptane::Target> getAvailableTargets(const PackageConfig& pcon
   const std::string search_msg{just_latest ? "a target" : "all targets"};
   LOG_INFO << "Searching for " << search_msg << " starting from " << allowed_targets.begin()->filename()
            << " that match content provided in the source directory\n"
-           << "\tpacman type: \t" << pconfig.type << "\n\t apps dir: \t" << src.AppsDir << "\n\t ostree dir: \t"
+           << "\t pacman type: \t" << pconfig.type << "\n\t apps dir: \t" << src.AppsDir << "\n\t ostree dir: \t"
            << src.OstreeRepoDir;
   for (const auto& t : allowed_targets) {
-    LOG_INFO << "Checking " << t.filename();
     if (!repo.hasCommit(t.sha256Hash())) {
-      LOG_INFO << "\tmissing ostree commit: " << t.sha256Hash();
+      LOG_DEBUG << "\t" << t.filename() << " - missing ostree commit: " << t.sha256Hash();
       continue;
     }
     if (pconfig.type != ComposeAppManager::Name) {
       auto custom_data{t.custom_data()};
       custom_data[LocalSrcDirKey]["ostree"] = src.OstreeRepoDir.string();
       found_targets.emplace_back(Target::updateCustom(t, custom_data));
-      LOG_INFO << "\tall target content have been found";
+      LOG_INFO << "\t" << t.filename() << " - all target content has been found";
```

Comment:
I probably should use something like `target elements` or `target components` instead of `content` since English grammar consider it as a singular noun :).